### ISSUE TITLE
Movies No Longer Playing

### ISSUE THREAD

**Issue by jguerinet** [streamaserver/streama#707]
### Issue description

The movies no longer play with 1.6.0-RC10. I believe this is because of #692. I have copies my comments on that PR below, and created an issue as @fuji44 has recommended. 

### Steps to Reproduce

1. Try playing a movie.

### Expected Behaviour

The movie should play.

### Actual Behaviour

Streama stays on the loading screen 

### Environment Information

- **Operating System**: Windows 10
- **Streama version**: 1.6.0-RC10

### Original Post
One small bug I noticed: this breaks Streama for non-TV shows. An exception is thrown because movies don't have an episode selector!

```
vendor.unminified.js:25497 TypeError: Cannot read property 'addEventListener' of null
    at J (streama.unminified.js:7833)
    at r (streama.unminified.js:7436)
    at streama.unminified.js:7420
    at vendor.unminified.js:23268
    at vendor.unminified.js:27811
    at r.$eval (vendor.unminified.js:29073)
    at r.$digest (vendor.unminified.js:28889)
    at r.$apply (vendor.unminified.js:29181)
    at e (vendor.unminified.js:23561)
    at v (vendor.unminified.js:23759)
```

Here's the code in question:
```
...
var playerMenu = document.querySelector('#player-menu-episode-selector');
playerMenu.addEventListener('mouseenter', function (event) {
    isMouseWheelVolumeCtrlActive = false;
});
...
```
I think this could be simply fixed with the following:
```
...
var playerMenu = document.querySelector('#player-menu-episode-selector');
if (playerMenu == null) {
    return;
}
playerMenu.addEventListener('mouseenter', function (event) {
    isMouseWheelVolumeCtrlActive = false;
});
...
```

**Comment by fuji44** [comment]
@jguerinet  Thank you for creating a ticket.
What do the browsers use?

**Comment by f1reflyyyylmao** [comment]
I have the same bug as described in the OP and can confirm with the following conditions:

Client:
Arch linux x64 / Windows 10 x64
Firefox stable & nightly

Server: 
Arch Linux x64 behind an nginx proxy
OpenJDK Runtime build 1.8.0_192-b26
Streama server 1.6.0-RC10

The file played is a .webm with vp8 video + vorbis cbr audio

**Comment by HannahRy** [comment]
This has been fixed in 382dfdb440a931dfc3ab2d87085be370b6b921a8.
